recipes/python-ftfy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-ftfy" %}
```

Comment:
That comes from the repository name: https://github.com/LuminosoInsight/python-ftfy

But I can change it to just `ftfy` if that's preferred.